.github/stale.yml

```
@@ -8,8 +8,9 @@ issues:
     daysUntilClose: 7
     # Issues with these labels will never be considered stale
     exemptLabels:
-      - Important
-      - Priority
+      - p0
+      - p1
+      - p2
```

Should we remove `p2` label from here?
```suggestion
      - p0
      - p1
```